src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +55,13 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString() {
+        String property = "Address : ";
+        if (!this.isPrivate) {
+            return property + this.toString();
+        }
+        else return "(Private)" + " " + this.toString();
```

Comment:
Coding standard violation! One more more of these problems detected: 
1. extra/missing spaces. 
2. brace style problem. 
3. missing braces.